README.md

```diff
@@ -0,0 +1,140 @@
+# FMS HF Tuning
+
+This repo aims to create basic tuning scripts with support for specific models, assumes the use of Hugging Face `SFTTrainer` and PyTorch FSDP only. Our approach to tuning is:
```

Comment:
some suggestions:
s/aims to create/provides a/
s/, assumes the use of/. The repo relies on/
s/ only//